engine/src/engine.rs

```diff
@@ -890,7 +896,27 @@ impl Engine {
             promise.args.as_slice(),
             promise.attached_balance,
             promise.attached_gas,
+        )
+    }
+
+    fn schedule_promise_callback(base_id: u64, promise: PromiseCreateArgs) -> u64 {
+        #[cfg(feature = "log")]
+        sdk::log_utf8(
+            crate::prelude::format!(
+                "[CALLBACK] Call contract: {}.{}",
```

Comment:
We should use a standard format for logs, much like we did in #323

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/311/commits/2635bff43b7a28444565c6c79df0f01fcf78aed4